tests/test_multi_thread.cpp

```diff
@@ -241,7 +240,7 @@ TEST_CASE("HNSW Multi-threading", "[ft][hnsw]") {
 
     float recall = correct / max_elements;
     std::cout << index->GetStats() << std::endl;
-    REQUIRE(recall == 1);
+    REQUIRE(recall > 0.96);
```

Comment:
Previously, it was not built concurrently, so the recall rate could be 1. However, a concurrent build cannot achieve such a high recall rate.